src/tests/ftest/control/dmg_telemetry_io_basic.yaml

```diff
@@ -0,0 +1,43 @@
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+  test_clients:
+    - client-A
+timeout: 150
+server_config:
+  name: daos_server
+  servers:
+    targets: 2
+pool:
+  name: daos_server
+  scm_size: 1G
+  control_method: dmg
+container:
+  type: POSIX
+  control_method: daos
+block_sizes: [10M, 500M]
+transfer_sizes: [1M]
+telemetry_metrics:
+  io_test_metrics: [
+    "engine_io_dtx_committed",
+    "engine_io_dtx_committed_max",
+    "engine_io_dtx_committed_mean",
+    "engine_io_dtx_committed_min",
+    "engine_io_dtx_committed_stddev",
+    "engine_io_ops_fetch_active_max",
+    "engine_io_ops_fetch_active_mean",
+    "engine_io_ops_fetch_active_min",
+    "engine_io_ops_fetch_active_stddev",
+    "engine_io_ops_update_active_max",
+    "engine_io_ops_update_active_mean",
+    "engine_io_ops_update_active_min",
+    "engine_io_ops_update_active_stddev"
+    ]
+  io_test_metrics_threshold: [0, 18446744073709552000]
```

Comment:
Question: Is the expected max threshold value for all engine_io* parameters supposed to be `18446744073709552000` or it should be different for different parameters?

Reply:
To check for this limit is after the fix DAOS-8128.